docs/modules/ROOT/pages/index.adoc

```diff
@@ -323,13 +325,36 @@ The `signer_payload` data structure MUST be presented to be signed by the _<<_cr
 
 IMPORTANT: If a data hash assertion is being used, the _<<C2PA claim generator>>_ MUST also follow the process described in xref:_interaction_with_data_hash_assertion[xrefstyle=full].
 
+==== Calculating the `expected_claim` value
+
+If the *<<_identity_assertion,identity assertion>> generator* chooses to include the `expected_claim` value in `signer_payload`, it MUST be computed as follows:
+
+. Create a valid _<<C2PA claim>>_ data structure that includes completed references to all _<<_c2pa_assertion,C2PA assertions>>,_ *including* all *<<_identity_assertion,identity assertion>>.*
+. For the specific *<<_identity_assertion,identity assertion>>* being computed, replace the `hash` value with 32 zero-value (`0x00`) bytes.
+. Serialize the _<<C2PA claim>>_ data structure using link:++https://www.rfc-editor.org/rfc/rfc8949.html#name-core-deterministic-encoding++[Section 4.2.1, “Core deterministic encoding,”] of RFC 8949: Concise binary object representation.
+. Compute a SHA-256 hash of the serialized _<<C2PA claim>>_ data structure.
+
+The resulting hash should be used as the `expected_claim` value.
+
+IMPORTANT: Since this algorithm replaces the `hash` value for exactly one *<<_identity_assertion,identity assertion>>,* this implies that only one *<<_identity_assertion,identity assertion>>* per _<<C2PA Manifest>>_ may provide an `expected_claim` value and that this *<<_identity_assertion,identity assertion>>* MAY NOT be referenced by any other *<<_identity_assertion,identity assertion>>.*
+
+==== Calculating the `expected_claim_generator` value
+
+If the *<<_identity_assertion,identity assertion>> generator* chooses to include the `expected_claim_generator` value in `signer_payload`, it MUST be computed as follows:
+
+. Identify signing certificate chain that will be used for the expected _<<C2PA claim generator>>._ This chain should include certificates for all intermediate certificate authorities.
+. Serialize this signing certificate chain as either a CBOR byte string (single certificate) or CBOR array of byte strings (certificate chain) as described in link:https://c2pa.org/specifications/specifications/2.0/specs/C2PA_Specification.html#x509_certificates[Section 14.6, “X.509 certificates,”] of the C2PA technical specification. This serialization should be performed using link:++https://www.rfc-editor.org/rfc/rfc8949.html#name-core-deterministic-encoding++[Section 4.2.1, “Core deterministic encoding,”] of RFC 8949: Concise binary object representation.
```

Comment:
For security, it should be sufficient to only sign the end-entity cert (because it's not possible to re-create a matching certificate from a different unexpected CA); an option, if that would simplify implementation...